connect/api/src/main/java/org/apache/kafka/connect/sink/SinkConnector.java

```diff
@@ -34,4 +34,12 @@
      */
     public static final String TOPICS_CONFIG = "topics";
 
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
```suggestion
```
we are not adding anything to the javadoc of the super method, so we shouldn't need `@inheritDoc`